etc/motd.template

```diff
@@ -2,8 +2,9 @@ Welcome to ravynOS __VERSION__ (__CODENAME__)!
 
 ** DEVELOPER PREVIEW RELEASE **
 
-Although airyxOS is mostly compatible with FreeBSD, please
+
+Although ravynOS is mostly compatible with FreeBSD, please
 do not report issues to the FreeBSD developers or bother the FreeBSD developers!
```

Comment:
OK - how about we change this to:
`do not bother the FreeBSD developers with ravynOS issue reports!`

Reply:
yes , that makes senese